### PR TITLE
Fix/fixing header

### PR DESCRIPTION
**Pull request by mmcode9407**

### CODE REVIEW

**Review by hansac91 — APPROVED** [review]

**Review by wawrzonnn — APPROVED** [review]
GJ

File: src/components/CreateProfileBtn/CreateProfileBtn.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 
 const CreateProfileBtn = () => {
-  const router = useRouter()
+  const { data: session } = useSession()
+
+  const onClickHandler = async () => {
+    if (!session) {
+      await signIn('github', {
+        callbackUrl: AppRoutes.githubOAuth,
+      })
```
Comment (Sumick) [inline src/components/CreateProfileBtn/CreateProfileBtn.tsx:15]: **issue:** user can be logged in without profile. we should `router.push` to create profile page, when user is logged in but does not have profile

**Review by Sumick — CHANGES_REQUESTED** [review]
Looks good, we need one important fix

**Review by Sumick — APPROVED** [review]
👌

**Pull request merged**